faq/static/faq/css/faqlist.css

```diff
@@ -0,0 +1,12 @@
+#outeraccordion{
+  width: 100%;
+  font-size: 150%;
+}
+
+.ui.accordion .title:not(.ui){
+  display: flex;
+}
+
+#pagetitle{
```

Comment:
Also, while you're at it, might as well rename this ID to `page-title` and `outeraccordion` above to `outer-accordion`, which is slightly easier to read 😊